forms/collections/elements.js

```diff
@@ -109,13 +116,14 @@ define(function (require) {
       };
 
       errors = this.reduce(function(memo, elementModel){
+        var subFormErrors;
         if ( elementModel.get('type') === 'subForm' ){
-          memo = _.chain(elementModel.get('forms').invoke('getInvalidElements'))
-                  .compact()
-                  .pluck('errors')
-                  .flatten()
-                  .reduce(reducer, memo)
-                  .value();
+          subFormErrors = _.chain(elementModel.get('forms').invoke('getInvalidElements'))
+                            .compact()
+                            .pluck('errors')
+                            .flatten()
+                            .value();
```

Comment:
Is it worth benchmarking this and seeing if it would be better to use a single optimised `for` loop instead of traversing the Objects / Array over and over with these functional methods?